bridge/engine/EngineMessageListener.h

```diff
@@ -1,16 +1,22 @@
 #pragma once
 
 #include "bridge/engine/EngineMessage.h"
+#include <string>
 
 namespace bridge
 {
+class Mixer;
+class EngineMixer;
 
 class EngineMessageListener
 {
 public:
     virtual ~EngineMessageListener() = default;
 
     virtual bool onMessage(EngineMessage::Message&& message) = 0;
+
+    virtual std::shared_ptr<Mixer> onEngineMixerRemoved1(EngineMixer& mixer) = 0;
```

Comment:
This is a bit of hack. We do not really want to abuse this interface like this.

Reply:
I think this methods shouldn't be part of `EngineMessageListener`.`EngineMixerRemoved` has direct access to `MixerManager` type.
Methods name with "1" and "2" are also not great.